app/src/main/java/com/tomtom/sdk/examples/usecase/BasicNavigationActivity.kt

```diff
@@ -148,14 +155,14 @@ class BasicNavigationActivity : AppCompatActivity() {
      * To use navigation in the application, start by by initialising the navigation configuration.
      */
     private fun initNavigation() {
-        val navigationConfiguration = NavigationConfiguration(
+        val configuration = Configuration(
             context = this,
             apiKey = apiKey,
             locationProvider = locationProvider,
-            routeReplanner = routeReplanner
+            routeReplanner = routeReplanner,
+            vehicleProvider = DefaultVehicleProvider(vehicle = Vehicle.Car())
```

Comment:
I think that's default value, maybe you can remove it.

Reply:
Technically it's true, but it was the default value event before this PR and in [previous PR](https://github.com/tomtom-international/tomtom-navigation-android-examples/pull/49) it was explicitly set for educational purposes I think.
@AdamRatcliffe-TomTom , what's your take on this? Should I revert it to the state where setting vehicle is done separately from creating Navigation?

Reply:
The vehicle provider was added explicitly because most of our SDK prospects are in the logistics space and doing truck routing and were setting the vehicle type to truck when planning the route without being aware they would need to specify the vehicle type when creating the navigation object in order to navigate as a truck.

Reply:
Is there value in setting vehicle type in Navigation separately from creating Navigation?

Reply:
If it can be set when creating navigation that's better.